plugins.py

```diff
@@ -0,0 +1,21 @@
+from courseware.tabs import EnrolledTab
+from django.conf import settings
+from django.utils.translation import ugettext_noop
+
+
+class NodeBBTab(EnrolledTab):
```

Comment:
What is this class doing?

Reply:
Responsible to create `tab` and contains some necessary `settings` related to tab like 
- name
- type
- title